src/Microsoft.AspNetCore.Razor.Tools/ServerCommand.cs

```diff
@@ -89,5 +105,40 @@ protected virtual void ExecuteServerCore(ConnectionHost host, CompilerHost compi
             var dispatcher = RequestDispatcher.Create(host, compilerHost, cancellationToken, eventBus, keepAlive);
             dispatcher.Run();
         }
+
+        internal FileStream WritePidFile()
+        {
+            // To make all the running rzc servers more discoverable, We want to write the process Id and pipe name to a file.
+            // The file contents will be in the following format,
+            //
+            // <PID>
+            // rzc
```

Comment:
The doc called for `razor`, but if you prefer `rzc`, I'll amend the doc (easy change on my side).

Reply:
`rzc` is the name of the server dll similar to `vbcscompiler`. So I thought this makes more sense.

Reply:
I'm good with this. I'll update the document and my change.